addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/LutronHandler.java

```diff
@@ -23,7 +24,7 @@
  * Base type for all Lutron thing handlers.
  *
  * @author Allan Tong - Initial contribution
- *
+ * @author Bob Adair - Added methods for status and state management
```

Comment:
Do you want to add the `@author` tag to every class you have changed in this PR?

Reply:
I guess I should. If only for purposes of allocating blame. Done.

Reply:
No, absolutely not. Feel free to ommit them, but do not mix it up.